### ISSUE TITLE
Is there an optimized version of SPTK for android platform?

### ISSUE THREAD

**Issue by attitudechunfeng** [r9y9/SPTK#22]

**Comment by r9y9** [comment]
No. However, SPTK is writter in C and I think it should work on android.

**Comment by attitudechunfeng** [comment]
You're right, it can well work on android. However, its speed is about 10 times slower than on PC, especially the post filter part when synthesizing audio. The speed of world on android is also very slow. This now confuses me.

**Comment by songmeixu** [comment]
@attitudechunfeng  Have you get a clue about how to use or optimized the SPTK or world on android?

**Comment by attitudechunfeng** [comment]
@songmeixu in fact， both SPTK and world can run on android for they're both written in c/c++. But the speed is somehow slower than PC. In my test, the main bottleneck of speed is world and model part. For world part, you can use multi threads. And for model part, you need a fast forward framework.
